Source/WebCore/html/PermissionsPolicy.cpp

```diff
@@ -280,6 +280,22 @@ static Ref<SecurityOrigin> declaredOrigin(const HTMLIFrameElement& iframe)
     return iframe.document().securityOrigin();
 }
 
+OwnerPermissionsPolicyData::OwnerPermissionsPolicyData()
+{
+    PermissionsPolicy permissionsPolicy;
+    // Align with default PermissionsPolicy value.
+    documentInheritedPolicyValue = permissionsPolicy.inheritedPolicyValue();
+}
+
+std::optional<HashSet<SecurityOriginData>> PermissionsPolicy::Allowlist::value() const
+{
+    return std::visit(WTF::makeVisitor([](const HashSet<SecurityOriginData>& origins) {
+        return std::optional<HashSet<SecurityOriginData>> { origins };
+    }, [&] (const auto&) {
+        return std::optional<HashSet<SecurityOriginData>> { };
+    }), m_origins);
```

Comment:
This copies everything. Maybe we should call it copyValue() or copyOrigins() instead.